src/locales/en-US/commands.json

```diff
@@ -34,53 +34,53 @@
     "clickHere": "Click here to invite me to your server",
     "noteThat": "Note that you need the **$t(permissions:MANAGE_GUILD)** permission in order to invite bots."
   },
   "ping": {
-    "commandDescription": "Checks if the bot is online"
+    "commandDescription": "Checks if the bot is online."
   },
   "github": {
-    "commandDescription": "Searches for a user, repository or organization on Github",
+    "commandDescription": "Searches for a user, repository or an organization on Github.",
     "commandUsage": "<type> <query>",
     "subcommands": {
       "user": {
-        "commandDescription": "Searches for an user on GitHub",
+        "commandDescription": "Searches for a user on GitHub.",
         "commandUsage": "<username>",
-        "noUser":  "You need to give me a valid user",
-        "userNotFound": "User not found",
-        "moreRepos": "...and {{repos}} more repositories",
-        "noBio": "This user doesn't have a biography",
+        "noUser":  "You need to give me a valid user!",
+        "userNotFound": "User not found.",
+        "moreRepos": "...and {{repos}} more repositories.",
+        "noBio": "This user doesn't have a biography.",
         "followers": "Followers",
         "following": "Following",
         "createdAt": "Account created at",
         "repositories": "Repositories ({{count}})"
       },
       "repository": {
-        "commandDescription": "Searches for a repository on GitHub",
+        "commandDescription": "Searches for a repository on GitHub.",
         "commandUsage": "<repository name>",
-        "noDescription": "This repository has no description",
+        "noDescription": "This repository has no description.",
         "watchers": "Watchers",
         "stars": "Stars",
         "forks": "Forks",
         "license": "License"
       },
       "organization": {
-        "commandDescription": "Searches for an organization on Github",
+        "commandDescription": "Searches for an organization on Github.",
         "commandUsage": "<organization name>",
-        "noOrg": "You need to give me a valid organization",
-        "orgNotFound": "Organization not found",
-        "moreRepos": "...and {{repos}} more repositories",
-        "noDescription": "This organization doesn't have a description",
+        "noOrg": "You need to give me a valid organization.",
+        "orgNotFound": "Organization not found.",
+        "moreRepos": "...and {{repos}} more repositories.",
+        "noDescription": "This organization doesn't have a description.",
         "members": "Members",
         "createdAt": "Organization created at",
         "repositories": "Repositories ({{count}})"
       }
     }
   },
   "spotify": {
-    "commandDescription": "Searches for a song, album, artist or a playlist on Spotify, or displays a user info.",
+    "commandDescription": "Searches for a track, playlist, an album or an artist on Spotify, or displays a user info.",
```

Comment:
```suggestion
    "commandDescription": "Searches for a track, playlist, album or artist on Spotify, or displays information about a user.",
```

Reply:
Try reading it out loud. My theory is when people read these messages they are reading like:
`"Searches for a track, a playlist, an album or an artist on Spotify, or displays a user info."`
So words starting with `a` and `an` are in separate groups in the sentence, so they are grammatically correct.